utils/util.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
```

Comment:
@gshipra What is the purpose of util.py? What problem does it solve?

Reply:
util.py has the code which can be reused by multiple scripts. Such as shutdown functionality. I did not want to duplicate the code for node shutdown functionality thus made this module.